pytest/tests/mocknet/mirror.py

```diff
@@ -423,6 +424,12 @@ def update_binaries_cmd(args, traffic_generator, nodes):
         help=
         'Starts all nodes and starts neard mirror run on the traffic generator.'
     )
+    start_traffic_parser.add_argument(
+        '--send-interval-millis',
+        type=int,
+        help=
+        'interval in millis between sending each mainnet block\'s worth of transactions'
```

Comment:
can you also add to the help what is default (what happens if this is not given)?

Reply:
added